pkg/util/utils.go

```diff
@@ -69,6 +69,32 @@ func StringInSlice(s string, sl []string) bool {
 	return false
 }
 
+// ParseChanges returns key, value(s) pair for given option.
+func ParseChanges(option string) (key string, vals []string) {
+	// Supported format as below
+	// 1. key=value
+	// 2. key value
+	// 3. key ["value","value1"]
```

Comment:
I prefer using a regex which is less prone to errors than a manual parsing approach. Especially the limitation of not allowing whitespaces between items (see case 3) is something we can easily support with regexes but is hard to get right manually.

Reply:
I looked into a regex for case 3) and came up with `(?:\[?"([^"]+)"\s*,?\s*\]?)`. Note that it only parses the value/list and not the key.

A go example with this regex: https://play.golang.org/p/o43IB2hIQXh

Reply:
Can we do the regex fix in a different PR.  I would like to get this in before the release.

Reply:
I am okay with adding a regex after the PR but we need more tests for it. Currently, we require specific input formats but don't return errors if the input is malformed. I don't want to merge before that's addressed.

If we consider this PR as bug fix, it can easily get into 1.6.1.

Reply:
I am trying to comeup with regex, but it will agin be mix of manual parsing + regex as each 3 use case are different.(I am not regex expert).

It will be nice, if we can add regex in next PR.

Reply:
@kunalkushwaha, thanks. Yes, let's add regexes later but I really want the function to return an error if the input does not have the required format.

Reply:
I will check each string for validity + brackets

Reply:
@vrothberg PTAL